src/Illuminate/Database/Query/Builder.php

```diff
@@ -1658,6 +1658,11 @@ public function aggregate($function, $columns = array('*'))
 	 */
 	public function insert(array $values)
 	{
+		if (empty($values))
+		{
+			return true;
```

Comment:
I thought whether it should return `true` or `false`. Returned boolean is success indicator. We can think what is success here - lack of error or insertion performed. I assumed that no error means success, because it did what we wanted and by empty array provided, we wanted to insert nothing.